src/Features/Core/Portable/InheritanceMargin/IRemoteInheritanceMarginService.cs

```diff
@@ -12,18 +12,11 @@ namespace Microsoft.CodeAnalysis.InheritanceMargin
 {
     internal interface IRemoteInheritanceMarginService
     {
-        ValueTask<ImmutableArray<InheritanceMarginItem>> GetGlobalImportItemsAsync(
+        ValueTask<ImmutableArray<InheritanceMarginItem>> GetInheritanceMarginItemsAsync(
```

Comment:
This is the core part of the change.
For both features, just send a document and text span to the OOP, and do all the semantics work there.
So we only need one method in the remote interface.